oclint-driver/reporters.h

```diff
@@ -3,4 +3,4 @@
 #include "oclint/Reporter.h"
 
 void loadReporter();
-oclint::Reporter* reporter();
+std::vector<oclint::Reporter*> reporter();
```

Comment:
Can you make this plural: `reporters`? I think that will also fix your CI failure.

Reply:
Thanks for your advice, Oscar. Have fix the problem as you suggested.